src/main/java/com/artipie/scripting/PythonScript.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.scripting;
+
+/**
+ * Python Script.
+ *
+ * @since 0.30
+ */
+public interface PythonScript {
```

Comment:
do not clear, why scripts for languages are interfaces... It's would be more logical to create parent class with two not static methods and with language name field which would be overridden in each accessor. Each accessor will implement only ctor, passing corresponding language name.